### PR TITLE
Add etcd and image configuration

### PR DESCRIPTION
**Pull request by eberlep**

### CODE REVIEW

File: pkg/lbmanager/lbmanager.go [hunk 1]
```diff
@@ -11,21 +11,23 @@ import (
...
+type Options struct {
 	LBIP           string
 	PortRangeStart int32
 	PortRangeSize  int32
 }
 
+// LBManager Responsible for the creation and deletion of externally accessible Services to access the Postgresql clusters managed by the Postgreslet.
+type LBManager struct {
+	client.Client
+	options *Options
```
Comment (Gerrit91) [inline pkg/lbmanager/lbmanager.go:23]: Maybe consider not passing a pointer here. Your code accesses the fields in this struct without checking for nil.

File: pkg/operatormanager/operatormanager.go [hunk 2]
```diff
@@ -353,6 +362,12 @@ func (m *OperatorManager) createNewClientObject(ctx context.Context, obj client.
 		}
 	case *appsv1.Deployment:
 		m.log.Info("handling Deployment")
+		if len(v.Spec.Template.Spec.Containers) != 1 {
+			m.log.Info("Unexpected number of containers in deployment, ignoring.")
+		} else if m.options.OperatorImage != "" {
+			m.log.Info("Patching operator image", "image", m.options.OperatorImage)
+			v.Spec.Template.Spec.Containers[0].Image = m.options.OperatorImage
```
Comment (Gerrit91) [inline pkg/operatormanager/operatormanager.go:369]: Is this kind of a new feature? What does this do?

**Review by Gerrit91 — COMMENTED** [review]
Looks nice and clean to me but I have left two questions that interest me.

File: pkg/lbmanager/lbmanager.go (hunk 1 above)
Comment (eberlep) [reply pkg/lbmanager/lbmanager.go:23]: True. And it's only passed alone once anyway.

File: pkg/operatormanager/operatormanager.go (hunk 2 above)
Comment (eberlep) [reply pkg/operatormanager/operatormanager.go:369]: Well... in this code, we parse the [YAML file with the complete operator deployment](https://github.com/fi-ts/postgreslet/blob/main/external/svc-postgres-operator.yaml) and modify it on the fly. This eases the maintenance of the deployment (we only need to update one file) and also makes our changes more visible.

In this specific case, I overwrite the [image](https://github.com/fi-ts/postgreslet/blob/main/external/svc-postgres-operator.yaml#L326) value in the deployment with the one configured in the postgreslet. This is a new feature. There is only one deployment contained in the YAML currently, so no special care is taken for finding that deployment, but the number of containers might vary in the future, so there is a little bit of a check in here.

File: pkg/lbmanager/lbmanager.go (hunk 1 above)
Comment (eberlep) [reply pkg/lbmanager/lbmanager.go]: Done.

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**